src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
and if u wanna help out and verify the babel plugin with `import()` doesn't work server-side, then try it with and without the babel plugin in an isolated context and see if `import().then` can return what you want.

if it works without the babel plugin, but doesnt with the babel plugin, then we know something is wrong.